uiowa/uiowa
# Removed aggregator library

Closes #4340 


# How to test


1. blt ds --site=admissions.uiowa.edu
2. inspect page -> console -> make sure doesn't display error
3. also make sure aggregators still look fly... even though library doesn't exist but if want to go that extra mile why not haha 😆
